packages/components/src/locales/en-us/messages.json

```diff
@@ -686,5 +686,20 @@
     "message": "Show Language Selector",
     "description": "Tooltip for Show Languges Titles Block",
     "component": "Grammar"
+  },
+  "LOOKUP_USE_BETA_CODES_CHECK": {
+    "message": "update beta codes",
+    "description": "Update beta codes to greek chars",
+    "component": "Lookup"
+  },
+  "LOOKUP_USE_BETA_CODES_INFO_FIRST": {
+    "message": "Type in unicode polytonic Greek using beta code as indicated below:",
```

Comment:
Please change the text to "You can type Greek using the following betacode characters. They will be converted upon input to Unicode."

Reply:
fixed